woocommerce/woocommerce-gateway-stripe
# Create a placeholder onboarding wizard page for UPE

# Changes proposed in this Pull Request:
Create a placeholder onboarding wizard page for UPE. This will only show when `_wcstripe_feature_upe_settings` feature flag is enabled. 

For this onboarding page, I used the `wc_admin_register_page()` to register it as a React-powered page. 

ref: 
- https://developer.woocommerce.com/extension-developer-guide/working-with-woocommerce-admin-pages/#registering-a-react-powered-page
- https://developer.woocommerce.com/2020/11/13/tutorial-adding-react-support-to-a-woocommerce-extension/


# Testing instructions
1. Make sure `_wcstripe_feature_upe_settings` is set to `1`; or have [this function](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/4d41eafca57d1ad7182b18f3ec26c6f860c44056/includes/class-wc-stripe-feature-flags.php#L22) return `true`.
2. Go to `wp-admin/admin.php?page=wc_stripe-onboarding_wizard` and you should see a placeholder onboarding wizard page. 
![image](https://user-images.githubusercontent.com/572862/130664961-8cc15eee-088e-4f81-a92f-3b6fe47d94aa.png)
